localgovdrupal/localgov_project
# Add dropzonejs composer merge

See https://github.com/localgovdrupal/localgov/pull/729



## What does this change?

Adds the wikimedia composer merge plugin, and then uses that to load dropzonejs composer.libraries.json to require the dropzone JS library.

## How to test

Composer install or composer update --lock should install the dropzonejs libary once that module is installed.

## How can we measure success?

Module gets installed

## Have we considered potential risks?

Others may want to have their own dropzone library, so will have to remove this.
Those who have created localgov drupal projects already will need to copy this step.

## Images

n/a

## Accessibility

n/a
